src/services/connection/connection.transport.ts

```diff
@@ -0,0 +1,6 @@
+export class ConnectionTransport {
```

Comment:
Also make sure to test your ConnectionTransports & Connectiontransport object. You can use 
connection.statuses.spec.ts & connection.status.spec.ts as an example.